lucene/test-framework/src/java/org/apache/lucene/index/BaseDocValuesFormatTestCase.java

```diff
@@ -1216,7 +1219,7 @@ private void doTestNumericsVsStoredFields(double density, LongSupplier longs) th
     doc.add(dvField);
     
     // index some docs
-    int numDocs = atLeast(300);
+    int numDocs = atLeast((int) (minDocs*1.172));
```

Comment:
why this 1.172 factor?

Reply:
Yeah, that should have been explained. It is because I made the minDocs variable. Originally the test was fixed with 256 as minDocs. `256*1.172=300`, so with this trick the `numDocs/minDocs`-factor stayed constant for the old tests. I have probably been overthinking this. Should I set it to something less puzzling like 1.5 or 2?